railties/lib/rails/application/configuration.rb

```diff
@@ -278,6 +278,7 @@ def load_defaults(target_version)
           if respond_to?(:active_support)
             active_support.default_message_encryptor_serializer = :json
             active_support.default_message_verifier_serializer = :json
+            active_support.cache_store_pooling_enabled_by_default = true
```

Comment:
If this is the default for new apps, should we add `connection_pool` to their gemfile?

Either that, or changing the comment @ https://github.com/rails/rails/blob/4732514bb0af535bc7209a67f3823b2ebcb52848/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L67

Reply:
> If this is the default for new apps, should we add `connection_pool` to their gemfile?

I don't think so as you only need it for memcached and redis stores.

Reply:
Right, and the `environments/production.rb` template invites you to use the memcache store. If you uncomment that line and don't read the docs closely enough, your app would crash on boot (because `connection_pool` is missing) only in production. I don't know if this warrants adding the gem but I think we could at least make the template file warn you better.